### PR TITLE
InstructorCourseEditPageUiTest: Some parts do not test the correct things #5623

### LINKED ISSUE #5623: InstructorCourseEditPageUiTest: Some parts do not test the correct things
Line 115 to 118 should be testing the access level view details links of the newly created instructor, but instead it is tested for the instructor currently being edited

Line 133 and 135 should be testing for the instructor currently being edited, but it tests for the added instructor.

### PR DESCRIPTION
**Pull request by JosephineKwa**
Fixes #5623

### CODE REVIEW

File: src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java [hunk 1]
```diff
@@ -232,8 +232,10 @@ public boolean clickShowNewInstructorFormButton() {
         return isFormShownCorrectly;
     }
     
-    public boolean clickOnAccessLevelViewDetails(String role) {
-        WebElement viewDetailsLink = browser.driver.findElement(By.cssSelector(
+    public boolean clickOnAccessLevelViewDetails(String formId, String role) {
```
Comment (YongJieYongJie) [inline src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java]: I'm not sure if passing in a CSS id is the usual way of doing things. The more common form is passing in an integer index (which will require refactoring to a sentinel value for new instructor), or if the element is unique enough, there will be a `@FindBy` WebElement.

Comment (YongJieYongJie) [inline src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java]: Then again this file is in a huge state of disarray (you know something is fishy when you `assertTrue` on a method that clicks something), maybe create an issue _specifying_ what refactoring has to be done.

**Comment by YongJieYongJie** [comment]
@JosephineKwa Some comments added

**Comment by JosephineKwa** [comment]
Ready for review

**Pull request merged**